pmm/pmm2-server-autobuild.groovy

```diff
@@ -106,13 +106,14 @@ pipeline {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh '''
                         sg docker -c "
+                            set -o errexit
+                            set -o xtrace
+
                             export PATH=$PATH:$(pwd -P)/build/bin
 
                             # 1st-party
                             build-server-rpm percona-dashboards grafana-dashboards
-                            build-server-rpm pmm-manage
```

Comment:
@Sudokamikaze please don't forget to include that in some other PR

Reply:
@Sudokamikaze ^^^ Do we have a ticket to track that?

Reply:
Lemme create one

Reply:
@AlekSi https://jira.percona.com/browse/PMM-6621